search/util.go

```diff
@@ -118,3 +122,13 @@ func RecordSearchCost(ctx context.Context,
 		}
 	}
 }
+
+const GeoBufferPoolCallbackKey = "_geo_buffer_pool_callback_key"
+
+// Assigning the size of the largest buffer in the pool to 24KB and
+// the smallest buffer to 24 bytes. The pools are used to read a
+// sequence of vertices which are always 24 bytes each.
+const MaxBufPoolSize = 24 * 1024
+const MinBufPoolSize = 24
```

Comment:
Update these names to better reflect what they're intended for - so minGeoBufPoolSize and maxGeoBufPoolSize.